tests/resources/service-templates/tosca-simple-1.0/node-cellar/node-cellar.yaml

```diff
@@ -98,8 +98,8 @@ topology_template:
               #token: { get_property: [ HOST, flavor_name ] }
       interfaces:
         Maintenance:
-          enable: juju > charm.maintenance_on
-          disable: juju > charm.maintenance_off
+          enable: juju \> charm.maintenance_on
```

Comment:
I've put the backslash to escape it and disable it for now. :/ Since the Juju plugin is not installed, the tests that use this would fail otherwise.